README.md

```diff
@@ -343,6 +343,23 @@ $ pip install robber
 $ nosetests tests/
 ```
 
+#### Run test in multiple python versions with tox
+
+- Prerequisite
+You must have [pyenv](https://github.com/yyuu/pyenv), [virtualenv](https://github.com/yyuu/pyenv-virtualenv) installed. Install needed python versions (on `.python-version`):
+
+```
+pyenv install <PYTHON_VERSION>
+```
+- Install `tox`
+```
+pip install tox
+```
+- Run test:
+```
+tox
+```
+
```

Comment:
Need to add the instruction for creating the corresponding versions of python with `pyenv` and use it with `pyenv local`.